File: Python/twosum.py
class Solution:
	def twosum(self, nums, target):
		map={}
		for i in range(0,len(nums)):
				if nums[i] not in map:
					map[target-nums[i]]=i+1
				else:
					return map[nums[i]],i+1
		return -1,-1 

nums=[3,2,4]
target=6
Solution answer
Solution.twosum(answer,nums,target)

File: Ama_java/TwoSum.java
public class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        if (nums==null || nums.length<2)
            return null;
        int[] result = new int[2];
        HashMap<Integer, Integer> map = new HashMap<>();
        for (int i=0; i<nums.length; i++)
        {
            if (map.containsKey(nums[i]))
            {
                result[0]=map.get(nums[i]);
                result[1]=i;
                return result;
            }
            else
            {
                map.put(target-nums[i],i);
            }
        }
        return null;
    }
}